cms/datasets/models.py

```diff
@@ -87,7 +87,7 @@ def formatted_edition(self) -> str:
 
     @property
     def url_path(self) -> str:
-        return f"/datasets/{self.namespace}/editions/{self.edition}/versions/{self.version}"
+        return f"datasets/{self.namespace}"
```

Comment:
q: is this because we're using the public API which doesn't have the new paths? If so, perhaps leave the other one with TODO comment?

Reply:
This is changing because it's been agreed that for now the links should just direct to the dataset landing page, will addc a comment in code and on the ticket.

Reply:
A side effect of this is that now that the validation that checks for duplicated links between Manually added and Looked up datasets doesn't work as expected. 


https://github.com/ONSdigital/dis-wagtail/blob/40da75b9f34b2f9f4e4bc8a1fc5210cd5a326fac/cms/datasets/blocks.py#L51
^ In this dict the URLs that will act as keys will be different for the manually linked and looked up datasets - even though the Looked-up dataset will still show the version number in the admin UI. It could be worth specifying in the comment in the `clean` method that this part won't work until we revert that.

Reply:
There's some confusion around this because we didn't get a definitive answer on where the links for datasets should go, for the moment we've agreed to just link to the dataset landing page rather than the specific edition or version. However, I don't think we want to force manual dataset links to conform to this, they might be useful as a way of linking to a specific edition or version if required, and I don't think we want to try and block them from linking to two different editions or versions of the same dataset in case that is valid behaviour.